<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="SystemSetting"
           tests="3"
           failures="3"
           errors="0"
           skipped="0"
           time="18.671">
  <properties />
            <testcase name="input_test1" time="7.777" classname="com.webtest.SystemSetting.demo.product_test">
                <failure type="org.openqa.selenium.UnhandledAlertException"
                  message=": 
Build info: version: &apos;3.6.0&apos;, revision: &apos;6fbf3ec767&apos;, time: &apos;2017-09-27T16:15:40.131Z&apos;
System info: host: &apos;LIU&apos;, ip: &apos;169.254.28.25&apos;, os.name: &apos;Windows 8.1&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;6.3&apos;, java.version: &apos;1.8.0_51&apos;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{moz:profile=C:\Users\ASUS\AppData\Local\Temp\rust_mozprofile.59MLPOsHQ56F, rotatable=false, timeouts={implicit=0, pageLoad=300000, script=30000}, pageLoadStrategy=normal, moz:headless=false, platform=XP, moz:accessibilityChecks=false, acceptInsecureCerts=true, browserVersion=57.0, platformVersion=10.0, moz:processID=17652, browserName=firefox, javascriptEnabled=true, platformName=XP, moz:webdriverClick=false}]
Session ID: 593cff10-07e4-4bdc-9481-e99925537485"
              >
<![CDATA[org.openqa.selenium.UnhandledAlertException: : 
Build info: version: '3.6.0', revision: '6fbf3ec767', time: '2017-09-27T16:15:40.131Z'
System info: host: 'LIU', ip: '169.254.28.25', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_51'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{moz:profile=C:\Users\ASUS\AppData\Local\Temp\rust_mozprofile.59MLPOsHQ56F, rotatable=false, timeouts={implicit=0, pageLoad=300000, script=30000}, pageLoadStrategy=normal, moz:headless=false, platform=XP, moz:accessibilityChecks=false, acceptInsecureCerts=true, browserVersion=57.0, platformVersion=10.0, moz:processID=17652, browserName=firefox, javascriptEnabled=true, platformName=XP, moz:webdriverClick=false}]
Session ID: 593cff10-07e4-4bdc-9481-e99925537485
  org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:118)
  org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
  org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:164)
  org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
  org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:586)
  org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.defaultContent(RemoteWebDriver.java:962)
  com.webtest.core.WebDriverEngine.leaveFrame(WebDriverEngine.java:78)
  com.webtest.SystemSetting.demo.product_test.input_test1(product_test.java:33)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
  org.testng.SuiteRunner.run(SuiteRunner.java:364)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
  org.testng.TestNG.runSuites(TestNG.java:1049)
  org.testng.TestNG.run(TestNG.java:1017)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
            <testcase name="qingkong_test" time="3.021" classname="com.webtest.SystemSetting.demo.product_test">
                <failure type="org.openqa.selenium.NoSuchFrameException"
                  message="No frame element found by name or id rightMain
Build info: version: &apos;3.6.0&apos;, revision: &apos;6fbf3ec767&apos;, time: &apos;2017-09-27T16:15:40.131Z&apos;
System info: host: &apos;LIU&apos;, ip: &apos;169.254.28.25&apos;, os.name: &apos;Windows 8.1&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;6.3&apos;, java.version: &apos;1.8.0_51&apos;
Driver info: driver.version: unknown"
              >
<![CDATA[org.openqa.selenium.NoSuchFrameException: No frame element found by name or id rightMain
Build info: version: '3.6.0', revision: '6fbf3ec767', time: '2017-09-27T16:15:40.131Z'
System info: host: 'LIU', ip: '169.254.28.25', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_51'
Driver info: driver.version: unknown
  org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.frame(RemoteWebDriver.java:925)
  com.webtest.core.WebDriverEngine.enterFrame(WebDriverEngine.java:68)
  com.webtest.SystemSetting.demo.product_Action.input(product_Action.java:22)
  com.webtest.SystemSetting.demo.product_test.qingkong_test(product_test.java:39)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
  org.testng.SuiteRunner.run(SuiteRunner.java:364)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
  org.testng.TestNG.runSuites(TestNG.java:1049)
  org.testng.TestNG.run(TestNG.java:1017)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
            <testcase name="input_test" time="7.873" classname="com.webtest.SystemSetting.demo.product_test">
                <failure type="java.lang.AssertionError"
                  message="expected [true] but found [false]"
              >
<![CDATA[java.lang.AssertionError: expected [true] but found [false]
  org.testng.Assert.fail(Assert.java:96)
  org.testng.Assert.failNotEquals(Assert.java:776)
  org.testng.Assert.assertTrue(Assert.java:44)
  org.testng.Assert.assertTrue(Assert.java:54)
  com.webtest.SystemSetting.demo.product_test.input_test(product_test.java:24)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
  org.testng.SuiteRunner.run(SuiteRunner.java:364)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
  org.testng.TestNG.runSuites(TestNG.java:1049)
  org.testng.TestNG.run(TestNG.java:1017)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
